pkg/operator/resources/errors.go

```diff
@@ -61,3 +64,17 @@ func ErrorNoAvailableNodeComputeLimit(resource string, reqStr string, maxStr str
 		Message: message,
 	})
 }
+
+func ErrorAPIUsedByAPISplitter(apiSplitters []string) error {
+	return errors.WithStack(&errors.Error{
+		Kind:    ErrAPIUsedByAPISplitter,
+		Message: fmt.Sprintf("cannot delete api because it is used by the following %s: %s", strings.PluralS("APISplitter", len(apiSplitters)), strings.StrsSentence(apiSplitters, "")),
+	})
+}
+
+func ErrorNotDeployedAPIsAPISplitter(notDeployedAPIs []string) error {
+	return errors.WithStack(&errors.Error{
+		Kind:    ErrNotDeployedAPIsAPISplitter,
+		Message: fmt.Sprintf("unable to find apis specified in apisplitter: %s", strings.StrsSentence(notDeployedAPIs, "")),
```

Comment:
nit: `"unable to find apis specified in apisplitter: %s", strings.StrsSentence(notDeployedAPIs, "")` -> `"unable to find specified %s: %s", strings.PluralS("api", len(notDeployedAPIs), strings.StrsAnd(notDeployedAPIs, "")`